src/main/java/frc/robot/subsystems/Acquisition.java

```diff
@@ -14,17 +14,17 @@
 import frc.robot.constants.NeoMotorConstants;
 
 public class Acquisition extends SubsystemBase {
-    private static DigitalInput topLaser;
-    private static DigitalInput bottomLaser;
-    private static SparkFlex acquisitionMotor;
-    private static SparkLimitSwitch algaeSwitch;
+    private final DigitalInput topLaser;
+    private final DigitalInput bottomLaser;
+    private final SparkFlex acquisitionMotor;
+    private final SparkLimitSwitch algaeSwitch;
     public static Acquisition instance;
 
     private Acquisition() {
-        topLaser = new DigitalInput(AcquisitionConstants.kTopLaserChannel);
-        bottomLaser = new DigitalInput(AcquisitionConstants.kBottomLaserChannel);
-        acquisitionMotor = new SparkFlex(AcquisitionConstants.kMotorDeviceID, MotorType.kBrushless);
-        algaeSwitch = acquisitionMotor.getReverseLimitSwitch();
+        topLaser = new DigitalInput(AcquisitionConstants.kTopLaserPort);
+        bottomLaser = new DigitalInput(AcquisitionConstants.kBottomLaserPort);
+        acquisitionMotor = new SparkFlex(AcquisitionConstants.kAcquisitionMotorPort, MotorType.kBrushless);
```

Comment:
Still need to move these news to be with their var definition and not in the constructor